roles/setup_gitea/templates/sa_anyuid.j2

```diff
@@ -0,0 +1,5 @@
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: anyuid
+  namespace: "{{ sg_namespace }}"
```

Comment:
Missed line break at the end of the file

Reply:
Done